packages/es-components-via-theme/index.js

```diff
@@ -225,6 +225,28 @@ const theme = {
       iconName: 'no-symbol'
     }
   },
+  bannerStyles: {
+    success: {
+      bgColor: success100,
+      textColor: black
+    },
+    info: {
+      bgColor: info100,
+      textColor: black
+    },
+    warning: {
+      bgColor: warning100,
+      textColor: black
+    },
+    danger: {
+      bgColor: danger100,
+      textColor: black
+    },
+    advisor: {
+      bgColor: agent300,
+      textColor: black
+    }
+  },
```

Comment:
We're adding our own `bannerStyles`, instead of using the already existing `notificationsStyles`. 
It's using different color shades for each `type`

Reply:
Perhaps Austin can give his reasoning here on why the different color shades.

Reply:
I agree. In this case, I use this color shade because it's a different color than LightNotification and Notifications use.